src/content/docs/reference/core.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Membrane core library
+---
+
+Programs written in Membrane are mostly just vanilla TypeScript/JavaScript, but there are a handful of patterns to get familiar with.
```

Comment:
This intro should call out that this is the documentation of the types and functions available to each program via the `membrane` module.